pages/home/pages/community/components/shared/OkCommunityProfileActionButtons.vue

```diff
@@ -70,11 +75,30 @@
             return this.community.colorInvert.hex();
         }
 
+        get buttonCssStyle() {
+            let style = {
+                color: "black",
+                'margin-right': "10px"
+            };
+
+            if (!this.community) return style;
+
+            const activeTheme = this.$observables.activeTheme.value;
+            const themeColorIsCommunityColor = this.community.color.hex() === activeTheme.primaryColor.hex();
+
+            style["background-image"] = "none";
+            style["backgroundColor"] = themeColorIsCommunityColor ? activeTheme.primaryHighlightColor.hsl().string() : this.community.color.hex();
+            style["color"] = themeColorIsCommunityColor ? activeTheme.primaryInvertColor.hex() : this.community.colorInvert.hex();
+
+            return style;
+        }
+
         private onLoggedInUserChanged(loggedInUser: IUser) {
             this.canBanOrUnban = loggedInUser.canBanOrUnbanUsersInCommunity(this.community);
+            this.canCreatePost = loggedInUser.canCloseOrOpenPostInCommunity(this.community) //TODO change service to one where you can create post if you are a user of the community
```

Comment:
Why a TODO?, this validation is not correct and you are not even using it, the Community interface contains a method to know wether an user is a member or not of the community